app/src/main/res/layout/widget_media_controls.xml

```diff
@@ -128,7 +152,7 @@
                     android:padding="4dp"
                     android:scaleType="fitCenter"
                     android:src="@drawable/ic_prev_track"
-                    android:tint="@color/colorIcon" />
+                    app:tint="@color/colorIcon" />
```

Comment:
thanks for pointing this out - it was presenting as an error in android studio e.g.:

"Error:(59, 17) Must use `app:tint` instead of `android:tint`"

actually resetting these to `android:tint` fixes the dark theme icon colours:

<img src="https://i.imgur.com/zqjmguu.png" width=300px/>

maybe this is an explanation of why android studio shows errors, but it is beyond me: https://stackoverflow.com/questions/64256397/android-imageview-tint

Reply:
Looks like its complaining because `android:tint` doesnt work on API levels below 21 however our app has 21 as the minimum so it doesn't apply to us. Seems like we may be able to change `ImageView` to `AppCompatImageView` to get around it but we can look into that in another PR. The main thing is to make sure that the checks pass and the UI isn't impacted :)